modules/ossm-vs-istio.adoc

```diff
@@ -10,27 +10,44 @@ An installation of {ProductName} differs from upstream Istio community installat
 The current release of {ProductName} differs from the current upstream Istio community release in the following ways:
 
 [id="ossm-multi-tenant-install_{context}"]
-== Multi-tenant installations
+== {ProductName} control plane
 
+{ProductName} installs a multi-tenant control plane by default. You specify the projects that can access the {ProductShortName}, and isolate the {ProductShortName} from other control plane instances.
+
+[id="ossm-mt-vs-clusterwide_{context}"]
+=== Multi-tenancy in {ProductName} versus cluster-wide installations
+
+The main difference between a multi-tenant installation and a cluster-wide installation is the scope of privileges used by the control plane deployments, for example, Galley and Pilot. The components no longer use cluster-scoped Role Based Access Control (RBAC) `ClusterRoleBinding`, but rely on {ProductShortName} project-scoped RBAC `ServiceMeshRbacConfig`.
+
+Every project in the `members` list will have a `RoleBinding` for each service account associated with a control plane deployment and each control plane deployment will only watch those member projects. Each member project has a `maistra.io/member-of` label added to it, where the `member-of` value is the project containing the control plane installation.
+
+{ProductName} configures each member project to ensure network access between itself, the control plane, and other member projects. The exact configuration differs depending on how OpenShift software-defined networking (SDN) is configured. See xref:../../networking/openshift-sdn/about-openshift-sdn.adoc#about-openshift-sdn[About OpenShift SDN] for additional details.
+
+* *`NetworkPolicy`*: {ProductName} creates a `NetworkPolicy` resource in each member project allowing ingress to all pods from the other members and the control plane. If you remove a member from {ProductShortName}, this `NetworkPolicy` resource is deleted from the project.
```

Comment:
These bullets appear have no lead-in sentence. Add something like "If the OCP cluster is configured to use the SDN plugin:"

Reply:
Fixed.